### ISSUE TITLE
ECSP200109_Puestoabends

### ISSUE THREAD

**Issue by klausriede** [BioAcoustica/BioAcoustica#247]
extremely slow display, could not play the sound...

Issue on page https://bio.acousti.ca/content/ecsp200109puestoabends

**Comment by edwbaker** [comment]
Yes - this has been reported to the NHM team, just waiting for them.

**Comment by klausriede** [comment]
In any case, download worked perfect, much appreciated!


Klaus Riede Global REgister of Migratory Species www.groms.de<http://www.groms.de>


________________________________
